frontend-ssr/redux/exercise-submissions/types.ts

```diff
@@ -10,6 +10,8 @@ export interface Submission {
 }
 export interface SubmissionState {
     submissions: Submission[],
+    currentPage: number,
+    countPerPage: number,
```

Comment:
Although this is generally a good idea, let's keep it as simple as possible for now. No need for a configurable `countPerPage`